sources/odoo.conf

```diff
@@ -4,7 +4,7 @@
 ; --------------------
 ; change this ASAP!!!!
 ;---------------------
-admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+admin_passwd = _ADMIN_PASSWORD_
 
 ; set the default timezone for openerp interface
 timezone = Europe/Paris
```

Comment:
not sure technically what is the best option but I prefer not to have it in the code for sure
=> up to you :)